### PR TITLE
[Frontend][Torch] Check graph inputs match expected

### PR DESCRIPTION
**Pull request by jjohnson-arm**
* error/warn when missing/unused graph inputs

### CODE REVIEW

**Comment by jjohnson-arm** [comment]
Fix for: https://discuss.tvm.ai/t/pytorch-wrongly-defined-graph-input-names-cause-keyerror/5866

File: python/tvm/relay/frontend/pytorch.py [hunk 1]
```diff
@@ -902,6 +902,21 @@ def _report_missing_conversion(op_names):
         msg = "The following operators are not implemented: {}".format(missing)
         raise NotImplementedError(msg)
 
+def _check_input_names(graph, input_shapes):
+    """ Check the graph inputs match the inputs """
+    # remove self at the 0th arg
+    ir_inputs = _get_input_names(graph)[1:]
```
Comment (masahi) [inline python/tvm/relay/frontend/pytorch.py]: Passing script_module and using _get_graph_input_names would be simpler?

Comment (jjohnson-arm) [reply python/tvm/relay/frontend/pytorch.py]: Fair enough - though I was just trying to avoid the extra graph.copy().

**Comment by masahi** [comment]
Let's wait until the github bug is fixed

**Comment by masahi** [comment]
thanks @jjohnson-arm

**Pull request merged**